key/store_test.go

```diff
@@ -5,17 +5,23 @@ import (
 	"path"
 	"testing"
 
-	"github.com/drand/drand/common"
+	commonutils "github.com/drand/drand/common"
 
 	kyber "github.com/drand/kyber"
 	"github.com/drand/kyber/share"
 	"github.com/stretchr/testify/require"
 )
 
+// GetBeaconIDFromEnv read beacon id from an environmental variable.
+// It is used for testing purpose.
+func GetBeaconIDFromEnv() string {
+	return commonutils.GetCorrectBeaconID(os.Getenv("BEACON_ID"))
+}
```

Comment:
Looks like this function is duplicated at https://github.com/drand/drand/pull/979/files#diff-b46b017e5fb75a98f5771351ea2671963ab0bbc879425f2b4643b730dacaed2cR43?

Reply:
Yes, it's the easiest way to avoid a cyclic dependency issue. The alternative would be to create a dedicated `test/utils` package for just that function. I'm happy with both, not sure what's best.